From: SeongJae Park <sj@kernel.org>
Date: Thu, 3 Oct 2024 15:04:06 -0700
Subject: [PATCH] selftests/damon/_debugfs_common: hide file write error
 message from test_write_result()

test_write_result() is used by DAMON debugfs interface selftests to
check if valid/invalid writes to files of the interface success/fails as
expected.  File write error messages from expected failures are only
making the test output noisy.  Hide such expected error messages.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 tools/testing/selftests/damon/_debugfs_common.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/damon/_debugfs_common.sh b/tools/testing/selftests/damon/_debugfs_common.sh
index aa995516870b..26b48bdbad84 100644
--- a/tools/testing/selftests/damon/_debugfs_common.sh
+++ b/tools/testing/selftests/damon/_debugfs_common.sh
@@ -8,7 +8,7 @@ test_write_result() {
 	expect_reason=$4
 	expected=$5
 
-	echo "$content" > "$file"
+	echo "$content" > "$file" 2> /dev/null
 	if [ $? -ne "$expected" ]
 	then
 		echo "writing $content to $file doesn't return $expected"
-- 
2.39.5

